PICMI_Python/applied_fields.py

```diff
@@ -0,0 +1,118 @@
+"""Classes following the PICMI standard
+These should be the base classes for Python implementation of the PICMI standard
+"""
+import re
+
+from .base import _ClassWithInit
+
+# ---------------
+# Applied fields
+# ---------------
+
+
+class PICMI_ConstantAppliedField(_ClassWithInit):
+    """
+    Describes a constant applied field
+      - Ex: Constant Ex field (float) [V/m]
+      - Ey: Constant Ey field (float) [V/m]
+      - Ez: Constant Ez field (float) [V/m]
+      - Bx: Constant Bx field (float) [T]
+      - By: Constant By field (float) [T]
+      - Bz: Constant Bz field (float) [T]
+      - lower_bound=[None,None,None]: Lower bound of the region where the field is applied (vector) [m]
+      - upper_bound=[None,None,None]: Upper bound of the region where the field is applied (vector) [m]
+    """
+    def __init__(self, Ex=None, Ey=None, Ez=None, Bx=None, By=None, Bz=None,
+                 lower_bound=[None,None,None], upper_bound=[None,None,None],
+                 **kw):
+
+        self.Ex = Ex
+        self.Ey = Ey
+        self.Ez = Ez
+        self.Bx = Bx
+        self.By = By
+        self.Bz = Bz
+
+        self.lower_bound = lower_bound
+        self.upper_bound = upper_bound
+
+        self.handle_init(kw)
+
+
+class PICMI_AnalyticAppliedField(_ClassWithInit):
+    """
+    Describes an analytic applied field
+      - Ex_expression: Analytic expression describing Ex field (string) [V/m]
+      - Ey_expression: Analytic expression describing Ey field (string) [V/m]
+      - Ez_expression: Analytic expression describing Ez field (string) [V/m]
+      - Bx_expression: Analytic expression describing Bx field (string) [T]
+      - By_expression: Analytic expression describing By field (string) [T]
+      - Bz_expression: Analytic expression describing Bz field (string) [T]
+               Expressions should be in terms of the position and time, written as 'x', 'y', 'z', 't'.
+               Parameters can be used in the expression with the values given as additional keyword arguments.
+               Expression should relative to the lab frame.
```

Comment:
```suggestion
               Expression should be relative to the lab frame.
```

Reply:
Thanks for the fix! I'll do a separate commit since I'll also change it to "Expressions", plural.